src/app/course-page/components/search-control/search-control.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="search-block-wrapper">
   <div class="container">
-    <input [(ngModel)]="inputValue">
+    <input type="text" [(ngModel)]="inputValue" (ngModelChange)="onChange($event)"/>
```

Comment:
I suggest remove two-way binding on ngModel and make : 
[ngModel]= "inputValue" and (ngModelChange)="Onchange "  
- and in the function OnChange - you will do: inputValue = $event; modelChangeCallback.emit().

Reply:
Removed two-way binding